testutil-client/src/main/java/io/spine/testing/client/TestActorRequestFactory.java

```diff
@@ -50,24 +50,30 @@ public TestActorRequestFactory(UserId actor, ZoneId zoneId) {
         super(ActorRequestFactory
                       .newBuilder()
                       .setActor(actor)
-                      .setZoneOffset(idToZoneOffset(zoneId))
+                      .setZoneOffset(toOffset(zoneId))
                       .setZoneId(zoneId)
         );
     }
 
-    public TestActorRequestFactory(@Nullable TenantId tenantId,
-                                   UserId actor,
-                                   ZoneOffset zoneOffset,
-                                   ZoneId zoneId) {
+    public TestActorRequestFactory(@Nullable TenantId tenantId, UserId actor, ZoneId zoneId) {
         super(ActorRequestFactory
                       .newBuilder()
                       .setTenantId(tenantId)
                       .setActor(actor)
-                      .setZoneOffset(zoneOffset)
+                      .setZoneOffset(toOffset(zoneId))
                       .setZoneId(zoneId)
         );
     }
 
+    @VisibleForTesting
```

Comment:
Do we need to specify `@VisibleForTesting`?  The whole `TestActorRequestFactory` class is `@VisibleForTesting`.

Reply:
It's a remnant of previous step. I'll surely remove the annotation, but later we'll need to [move the method into the Time library](https://github.com/SpineEventEngine/time/issues/87).